MSAL/test/app/mac/MSALAcquireTokenViewController.m

```diff
@@ -72,6 +72,10 @@ @implementation MSALAcquireTokenViewController
 - (void)viewDidLoad
 {
     [super viewDidLoad];
+    //TODO: MSAL MacOS test app doesn't support embedded webview mode
+    //Also, webView is initialized from the storyboard.
+    //Need to create it programmatically in order to be able to initialize it with a default recommended config setting,
+    //which can be created by using WKWebViewConfiguration *defaultWKWebConfig = [MSALWebviewParameters defaultWKWebviewConfiguration];
```

Comment:
I think MSAL Mac test app does support passed in webview. Can you double check?

Reply:
It does. What I meant was the embedded webview mode that uses the default webview config that gets initialized when the webView is loaded within MSIDWebviewUIController/loadView within macOS target; passedIn view stops at line 83 and never makes it past it. But I agree that this comment is misleading, so I believe it would be better to be removed.  but the second paragraph still has a valid point I believe.

Reply:
Based on our off-line discussion, I will change the passed-in view's embedded webview object to be loaded programmatically instead of being pulled from the storyboard as a weak IBOutlet variable.